components/clp-py-utils/clp_py_utils/clp_config.py

```
@@ -666,23 +667,24 @@ def make_config_paths_absolute(self, clp_home: pathlib.Path):
         )
         self._version_file_path = make_config_path_absolute(clp_home, self._version_file_path)
 
-    def validate_logs_input_config(self):
+    def validate_logs_input_config(self, use_host_mount: bool = False):
```

Ruff (0.14.2)

Boolean-typed positional argument in function definition

(FBT001)